CONTRIBUTING.md

```diff
@@ -29,7 +29,7 @@ We’re especially interested in learning what you need to implement the [Standa
 
 ## Public organizations
 
-We’re looking for public members to help shape the Foundation for Public Code’s long term future. As a member, you’ll have the chance to shape the Foundation’s processes and governance. Find out more about the [benefits and requirements of becoming a member](https://publiccode.net/membership/) and get in touch with us at <membership@publiccode.net>.
+We’re looking for public partners to help shape the Foundation for Public Code’s long term future. Find out more about the [benefits and requirements of becoming a partner](organization/partnerships.md) and get in touch with us at <partnerships@publiccode.net>.
```

Comment:
The "find out more" phrasing is very external facing, which strikes me as a contrast to the major tone of "About" which is typically more of a manual or a How-To guide. Perhaps simply change this to a mention that we have a useful email alias and a "see also" to the partnerhip page.

Reply:
This is a sharp observation. I think we may have been a bit lazy in thinking where to put what. As an invitation to public organizations, info about partnership should live on publiccode.net. Here, on About, we should have the information needed for staff to process partnerships.

Reply:
This isn't lazy; this was deliberately chosen many years ago.
Currently, the first call to action from publiccode.net links to this contributing page, because of Boris's understanding of About as the essence of the organization, thus making the About contributing page the invitation for everyone to become immediately, substantively involved. (We also do not have a contributing page on publiccode.net, because we didn't expect non-staff to contribute to our awareness raising site, though people have).
This will almost certainly change with the new website.

@clausmullie and I also discussed where to put new, proactive partnership content specifically - in earlier iterations of this PR, we linked to a currently non-existent publiccode.net/partnerships, but chose to make this PR independent of other PRs so that it could be merged and the content subsequently updated.

Reply:
@clausmullie created #1347 in response to our discussions yesterday.

Reply:
Ah, if it was deliberate, it wasn't explicit enough to not confuse me, as I always found it odd. The landing page of About, especially the "Information and processes" section, makes it look like the site is targeting staff.

Reply:
Okay, great to see that issue. Yet, I'd suggest we do not add to the #1347 debt, rather create these pages as they should be, and later create the call-to-action kinds of pages on publiccode.net

Reply:
Great discussion! We should definitely refactor About soon. Look forward to doing that, and as this PR is about removing members, I propose we close this discussion for now, merge it with the current debt it has, and fix it as part of #1347 and new work on the .net site to make it inviting to collaborators.